docker/package/manylinux/Dockerfile

```diff
@@ -33,7 +33,8 @@ RUN yum-config-manager --add-repo https://yum.repos.intel.com/oneapi && \
     rpm --import https://yum.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB && \
     yum update -y && yum install -y epel-release && \
     yum -y install centos-release-scl && \
-    yum install -y intel-oneapi-mkl-devel-2021.2.0 nasm rdma-core-devel devtoolset-7-gcc* rsync gdb
+    yum install -y intel-oneapi-mkl-devel-2021.2.0 nasm rdma-core-devel devtoolset-7-gcc* rsync gdb && \
+    yum -y install cuda-drivers
```

Comment:
we don't install CUDA from distro package management because it is always outdated.

Reply:
we are not using this docker file now. This is mainly for reference. We should have removed it.

Reply:
where is the dockerfile?

Reply:
Where needs to be changed?

Reply:
> we are not using this docker file now. This is mainly for reference. We should have removed it.

Where needs to be changed?